web-app/HeimScripts/heatmap/preprocess.R

```diff
@@ -0,0 +1,47 @@
+library("WGCNA")
+
+
+main <- function(aggregate=FALSE){
+  msgs = c("")
+  df <- loaded_variables[[length(loaded_variables)]]
+  good.input <- ncol(df) > 3
+  if(aggregate && good.input){
+    df <- dropEmptyGene(df)
+    aggr  <- aggregate.probes(df)
+    assign("preprocessed", aggr, envir = .GlobalEnv)
+    Discarded.rows <- nrow(df) - nrow(aggr)
+    msgs <- c( paste("Total discarded rows:",Discarded.rows))
```

Comment:
c() around it is not necessary:
c( paste("Total discarded rows:","test"))==paste("Total discarded rows:","test")
[1] TRUE